.github/workflows/publish-canary.yml

```diff
@@ -72,9 +73,9 @@ jobs:
           node ops/scripts/ci-versions.js ${{ toJSON(steps.changesets.outputs.publishedPackages) }}
 
       - name: Docker Image Name
-        id: docker_image_name
+        id: docker-image-name
         run: |
-          if [ -z "${CUSTOM_IMAGE_NAME}" ]
+          if [ ${CUSTOM_IMAGE_NAME} == '' ]
           then
               echo "::set-output name=canary-docker-tag::${GITHUB_SHA::8}"
           else
```

Comment:
One nit is that having the `prerelease` as a prefix breaks semantic versioning, see https://semver.org/#spec-item-9